gym_xiangqi/piece.py

```diff
@@ -63,14 +63,20 @@ def check_action(piece, orig_pos, cur_pos, repeat, offset, i, state, actions):
     r = cur_pos[0]
     c = cur_pos[1]
 
+    if piece < 0:
```

Comment:
nit-picking, but should standardize the naming of `piece_id` as used in many other places.

Reply:
done

Reply:
done